cachi2/core/models/property_semantics.py

```diff
@@ -58,6 +60,8 @@ def from_properties(cls, props: Iterable[Property]) -> "Self":
                 pip_package_binary = True
             elif prop.name == "cachi2:bundler:package:binary":
                 bundler_package_binary = True
+            elif prop.name == "cachi2:rpm_summary":
+                rpm_summary = ""
```

Comment:
This line here will cause this feature to never actually format the property to the output. I'd suggest adding an integration test case for this just to be sure we're covered.

Reply:
Tangential nitpick - since this is a custom property only relevant to RPMs and only ever used with RPM components, do we need the additional slight redundancy in the the property name - "rpm_summary" instead of just "summary"?

Reply:
Oof, good catch, thank you!

> do we need the additional slight redundancy?

I believe yes, it slightly pollutes SBOMs, but at the same time makes it easier to comprehend where it belongs when reading Properties code.

Reply:
Added an integration test to cover this case.